util/hash/fuzz/Cargo.toml

```diff
@@ -0,0 +1,22 @@
+
+[package]
+name = "keccak-hash-fuzz"
+version = "0.0.1"
+authors = ["Automatically generated"]
```

Comment:
if it is automatically generated (which is not completely the case i guess), then no need to include it's source, generate on build, otherwise put your name there

Reply:
This was autogenerated when cargo-fuzz initialized the fuzzing crate. Updated to include my name, and Parity Technologies as authors.